baekjoon/듣보잡.js

```diff
@@ -0,0 +1,28 @@
+const input = 
+  require('fs')
+  .readFileSync('/dev/stdin')
+  .toString()
+  .split('\n');
+
+const nameObj = {};
+const result = [];
+let cnt = 0;
+
+for (let i = 1; i < input.length; i += 1) {
+  if (!(input[i] in nameObj)) nameObj[input[i]] = 1;
+  else nameObj[input[i]] += 1;
+}
```

Comment:
이렇게 삼항 연산자로 표현해주면 더 간결한 표현이 가능해요 :)
```suggestion
for (let i = 1; i < input.length; i += 1) {
  input[i] in nameObj ? nameObj[input[i]] += 1 : nameObj[input[i]] = 1;
}
```